views/peep_page.erb

```diff
@@ -1,4 +1,4 @@
-Your Peeps: 
+<h2 style="color:blue;">Have a look at your Peeps playa! </h2>
```

Comment:
Good use of styling, perhaps consider a refactor and pulling it out into a style.css file and then any h2 can be in blue